PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -17,9 +17,9 @@ var settings = {
 
 // don't change anything below this line, unless you know what you're doing
 var	url = WScript.Arguments(0),
-	match = /^phpstorm:\/\/open\?(url=file:\/\/|file=)(.+)&line=(\d+)$/.exec(url),
+	match = /^phpstorm:\/\/open\/\?(url=file:\/\/|file=)(.+)&line=(\d+)$/.exec(url),
 	project = '',
-	editor = '"C:\\' + (settings.x64 ? 'Program Files (x86)' : 'Program Files') + '\\JetBrains\\' + settings.folder_name + '\\bin\\PhpStorm.exe"';
+	editor = '"C:\\' + (settings.x64 ? 'Program Files (x86)' : 'Program Files') + '\\JetBrains\\' + settings.folder_name + '\\bin\\phpstorm.exe"';

```

Comment:
It's lowercase in my computer, see:

![p-2017-06-30-001](https://user-images.githubusercontent.com/1639206/27698132-c9c1f06a-5d28-11e7-8aa6-61db62ad208b.jpg)
![p-2017-06-30-002](https://user-images.githubusercontent.com/1639206/27698133-c9ca7f28-5d28-11e7-9057-7f1df614777e.jpg)

It seems phpstorm's binary name and folder name are different in everyone's computer, so I think we need more people's test.